src/routes.json

```diff
@@ -24,6 +24,18 @@
         "title": "Hour of Code"
     },
     {
+        "name": "explore",
+        "pattern": "^/explore/:projects/:all/?$",
```

Comment:
i think that makes sense – do we want to do something a bit more restrictive though? i.e. make it be alphanumeric, and make it be under a certain character limit? Or would that make the rule too large...